README.md

```diff
@@ -189,6 +190,26 @@ x_thread = @> begin
     @select(:meanX, :meanY, var = :b)
 end
 ```
+
+Please note that `Lazy.jl` exports the function `groupby` which would clash
```

Comment:
can you pleae remove `\`` in all places where a package like Lazy.jl is referred to (as it is a package name not a code)

Reply:
done